tools/wptrunner/wptrunner/browsers/chrome_android.py

```diff
@@ -182,6 +189,16 @@ def logcat_cmd(self):
         cmd.extend(['logcat', '*:D'])
         return cmd
 
+    def possible_parse_tomestone(self, logger):
```

Comment:
typo (tomestone should be tom*b*stone)
Also, consider renaming to "maybe_parse_tombstone" which I think is a more common naming pattern.